docs/training/hugging_face.rst

```diff
@@ -2,27 +2,25 @@
 Hugging Face Guide
 ==================
 
-Uploading and Downloading Datasets on Hugging Face
-==================================================
 
 Creating an Account
--------------------
+^^^^^^^^^^^^^^^^^^^
 
 If you don't already have an account, sign up for a new account on the `Hugging Face Sign Up <https://huggingface.co/join>`_.
 
 Creating a New Dataset Repository
----------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Web Interface
-^^^^^^^^^^^^^
+Web Interface (Repository)
```

Comment:
Why the differentiation between Repository and Dataset Upload? If you put these under sections, that should be enough to explain the differences

Reply:
Okay